elf2cfetbl_version.h

```diff
@@ -0,0 +1,46 @@
+/*
+**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**  Copyright (c) 2006-2019 United States Government as represented by
+**  the Administrator of the National Aeronautics and Space Administration.
+**  All Rights Reserved.
+**
+**  Licensed under the Apache License, Version 2.0 (the "License");
+**  you may not use this file except in compliance with the License.
+**  You may obtain a copy of the License at
+**
+**    http://www.apache.org/licenses/LICENSE-2.0
+**
+**  Unless required by applicable law or agreed to in writing, software
+**  distributed under the License is distributed on an "AS IS" BASIS,
+**  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**  See the License for the specific language governing permissions and
+**  limitations under the License.
+*/
+
+/*
+ *  File: elf2cfetbl_version.h
+ *
+ *  Purpose:
+ *  Provide version identifiers for the ELF to cFE Table Converter.
+ *  See cfe documentation for version and build number and description
```

Comment:
@skliper do you have a better way of referring to the doxygen docs?

Reply:
If it's part of a doxygen block you can create a reference to other elements like this.